src/commands/build.ts

```diff
@@ -26,6 +26,12 @@ import srcFileExtensionMapping from './src-file-extension-mapping';
 export async function command(commandOptions: CommandOptions) {
   const {cwd, config} = commandOptions;
 
+  const configSrcFileExtensionMapping = Object.assign(
+    {},
+    srcFileExtensionMapping,
+    config.mapping || {},
+  );
```

Comment:
This is perfect for creating a new object reference! But since we already have `deepmerge` as a dependency for this project (creates deep clones of objects), maybe use that here instead?

Reply:
Sure, I'll make an update